Casks/x/xppen-pentablet.rb

```diff
@@ -0,0 +1,47 @@
+cask "xppen-pentablet" do
+  version "4.0.8,250414"
+  sha256 "c792fc0c5df8c38b75267d302d0c3ced49371ac7be73998875279bc627dad3ca"
+
+  url "https://download01.xp-pen.com/file/20#{version.csv.second[0, 2]}/#{version.csv.second[2, 2]}/XPPenMac_#{version.csv.first}_#{version.csv.second}.zip"
+  name "XPPen PenTablet"
+  desc "Universal driver for XPPen drawing tablets and pen displays"
+  homepage "https://www.xp-pen.com/"
+
+  livecheck do
+    url "https://www.xp-pen.com/download/star-g640.html"
+    regex(/XPPenMac[._-]v?(\d+(?:\.\d+)+)[._-](\d+)/i)
+    strategy :page_match do |page, regex|
+      page.scan(regex).max_by { |match| match[1] }&.then { |match| "#{match[0]},#{match[1]}" }
```

Comment:
```suggestion
      page.scan(regex).map { |match| "#{match[0]},#{match[1]}" }
```

`brew livecheck` handles the sorting of the versions for us already.